devel/babashka/Portfile

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        babashka babashka 1.3.176 v
+revision            0
+fetch.type          git
```

Comment:
this shouldn't be needed; instead set the correct "github.tarball_from ???" and add the checksums